diff_drive_controller/src/speed_limiter.cpp

```diff
@@ -69,6 +71,8 @@ namespace diff_drive_controller
   , max_acceleration(max_acceleration)
   , min_jerk(min_jerk)
   , max_jerk(max_jerk)
+  , min_deceleration(min_deceleration)
```

Comment:
move them up, just below the `_acceleration` ones (here also keeps the order of declaration, which should silence some compiler warnings)